### PR TITLE
created 2 endpoints for the location service

### PR DESCRIPTION
**Pull request by TooFatToKidnapp**
created 2 endpoints:
/countries : lists all countries with pagination parameter, returns the countryCode. 
/countries/:countryCode: list all cities in specified country pagination parameter

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -16,6 +16,6 @@ jobs:
       - name: docker login
         run: "docker login -u hyper -p ${{ secrets.DOCKER_REGISTRY_PASSWORD }} registry.hyper.mathematikoi.agency"
       - name: build image
-        run: "docker build -t registry.hyper.mathematikoi.agency/new.service.hyper:${{  github.ref_name }} ."
+        run: "docker build -t registry.hyper.mathematikoi.agency/location.service.hyper.service.hyper:${{  github.ref_name }} ."
```
Comment (mathematikoi) [inline .github/workflows/build.yaml:19]: the name seems wrong here

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -16,6 +16,6 @@ jobs:
...
       - name: push image
-        run: "docker push registry.hyper.mathematikoi.agency/new.service.hyper:${{  github.ref_name }}"
+        run: "docker push registry.hyper.mathematikoi.agency/location.service.hyper.service.hyper:${{  github.ref_name }}"
```
Comment (mathematikoi) [inline .github/workflows/build.yaml:21]: the name seems wrong here

File: chart/Chart.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: new-service-hyper
+name: location.service.hyper-service-hyper
```
Comment (mathematikoi) [inline chart/Chart.yaml:2]: the name seems wrong here

File: chart/templates/configmap.yaml [hunk 4]
```diff
@@ -2,10 +2,10 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ .Release.Name }}-new-service-hyper-configmap
+  name: {{ .Release.Name }}-location.service.hyper-service-hyper-configmap
```
Comment (mathematikoi) [inline chart/templates/configmap.yaml:5]: the name seems wrong here

File: chart/templates/deployment.yaml [hunk 5]
```diff
@@ -1,23 +1,23 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ .Release.Name }}-new-service-hyper-deployment
+  name: {{ .Release.Name }}-location.service.hyper-service-hyper-deployment
```
Comment (mathematikoi) [inline chart/templates/deployment.yaml:4]: the name seems wrong here

**Review by mathematikoi — CHANGES_REQUESTED** [review]

**Comment by TooFatToKidnapp** [comment]
@mathematikoi if a api endpoint responds with a array of objects as one of its fields, what is the appropriate definition of the responses  properties ? should i express that the values type is a array ? or should i be more verbose and express the types of every field in the array is the case of a object?.

**Comment by mathematikoi** [comment]
> @mathematikoi if a api endpoint responds with a array of objects as one of its fields, what is the appropriate definition of the responses properties ? should i express that the values type is a array ? or should i be more verbose and express the types of every field in the array is the case of a object?.

it actually depends, we need documentation to what a service should respond with.

**Pull request closed without merging**